src/app/(portal)/_components/footer/__tests__/Footer.component.test.tsx

```
@@ -0,0 +1,77 @@
+import { render, screen } from '@testing-library/react';
+
+import {
+  EMAIL_URL,
+  GITHUB_PROFILE_URL,
+  LINKEDIN_PROFILE_URL,
+} from '@/shared/constants/links.constants';
+
+import Footer from '../Footer.component';
```

ESLint

[error] '../Footer.component' import is restricted from being used by a pattern.

(no-restricted-imports)